.travis.yml

```diff
@@ -9,27 +9,34 @@ services:
 env:
   global:
     - REMOTE_IMAGE="ezsystems/php"
-    - LATEST="7.2"
-    - FORMAT_VERSION="v1"
+    - LATEST_PHP="7.3"
+    - LATEST_NODE="12"
+    - FORMAT_VERSION="v2"
   matrix:
-    # Run once per Dockerfile-<PHP_VERSION>
-    - PHP_VERSION="5.6" XDEBUG_CHANNEL="xdebug-2.5.5" EZ_VERSION="^1.13.4"
-    - PHP_VERSION="7.0" XDEBUG_CHANNEL="xdebug-2.8.1" EZ_VERSION="^1.13.4"
-    - PHP_VERSION="7.1"
-    - PHP_VERSION="7.2"
-    - PHP_VERSION="7.3"
+    # Run per Dockerfile-<PHP_VERSION>-<NODE_VERSION>
+    - PHP_VERSION="7.1" NODE_VERSION="10" EZ_VERSION="dev-adapt-to-docker-images-v2"
+    - PHP_VERSION="7.1" NODE_VERSION="12" EZ_VERSION="dev-adapt-to-docker-images-v2"
+    - PHP_VERSION="7.1" NODE_VERSION="14" EZ_VERSION="dev-adapt-to-docker-images-v2"
+    - PHP_VERSION="7.2" NODE_VERSION="10" EZ_VERSION="dev-adapt-to-docker-images-v2"
+    - PHP_VERSION="7.2" NODE_VERSION="12" EZ_VERSION="dev-adapt-to-docker-images-v2"
+    - PHP_VERSION="7.2" NODE_VERSION="14" EZ_VERSION="dev-adapt-to-docker-images-v2"
+    - PHP_VERSION="7.3" NODE_VERSION="10" UPDATE_PACKAGES="1" EZ_VERSION="dev-adapt-docker-to-format2-v3"
+    - PHP_VERSION="7.3" NODE_VERSION="12" UPDATE_PACKAGES="1" EZ_VERSION="dev-adapt-docker-to-format2-v3"
+    - PHP_VERSION="7.3" NODE_VERSION="14" UPDATE_PACKAGES="1" EZ_VERSION="dev-adapt-docker-to-format2-v3"
     # As builds are made with minimum supported PHP versions we need to force composer update for PHP 7.4 due to ocramius/proxy-manager
-    - PHP_VERSION="7.4" UPDATE_PACKAGES="1"
+    - PHP_VERSION="7.4" NODE_VERSION="10" UPDATE_PACKAGES="1" EZ_VERSION="dev-adapt-docker-to-format2-v3"
+    - PHP_VERSION="7.4" NODE_VERSION="12" UPDATE_PACKAGES="1" EZ_VERSION="dev-adapt-docker-to-format2-v3"
+    - PHP_VERSION="7.4" NODE_VERSION="14" UPDATE_PACKAGES="1" EZ_VERSION="dev-adapt-docker-to-format2-v3"
 
 before_script:
   - if [ ! -d ~/.composer/ ] ; then mkdir ~/.composer/; fi
   - echo "{\"github-oauth\":{\"github.com\":\"d0285ed5c8644f30547572ead2ed897431c1fc09\"}}" > ~/.composer/auth.json
   - bin/.travis/update_docker.sh
-  - bin/.travis/build.sh ${PHP_VERSION}
+  - bin/.travis/build.sh ${PHP_VERSION} ${NODE_VERSION}
 
 script: bin/.travis/test.sh
 
-after_success: if [ "$TRAVIS_BRANCH" = "v1" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bin/.travis/push.sh ${REMOTE_IMAGE} ${FORMAT_VERSION}; fi
+after_success: if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bin/.travis/push.sh ${REMOTE_IMAGE} ${FORMAT_VERSION}; fi
```

Comment:
>  "$TRAVIS_BRANCH" = "master" 

This is correct, but do we want to have it explit here that we push both master and v1 to make it more clear what we push for now?